tests/qos/qos_sai_base.py

```diff
@@ -2267,6 +2267,16 @@ def skip_pacific_dst_asic(self, dutConfig):
         yield
         return
 
+    @pytest.fixture(scope="function", autouse=False)
+    def skip_longlink(self, dutQosConfig):
```

Comment:
@zhixzhu 
pre-commit check failed, as below:

Pre-commit check results:
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check yaml...............................................................Passed
check for added large files..............................................Passed
check python ast.........................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

tests/qos/qos_sai_base.py:2317:5: F811 redefinition of unused 'skip_longlink' from line 2271

Reply:
@XuChen-MSFT  This PR https://github.com/mssonicbld/sonic-mgmt/pull/1 is to fix pre-commit failure, I don't have write access of mssonicbld:cherry/202311/10838

Reply:
Hi @zhixzhu better re-create manually double-commit for https://github.com/sonic-net/sonic-mgmt/pull/10838, rather than create an additional patch for this automate PR.

Reply:
PR to double commit: https://github.com/sonic-net/sonic-mgmt/pull/14584